### DISCUSSION TITLE
Change after updating template one

### DISCUSSION THREAD

**Discussion by patrickus33** [templaza/astroid-framework#784] [General]
After updating template one, the layout of the articles has changed.
The “shadow p-4 rounded” css classes were added without any action on my part.
How can I remove them?
This CSS appears on all “blog” views

for the moment, I'm bypassing the problem with CSS, but I hope this is a temporary solution and that the bug will be corrected.

You can see it here :
https://www.mairie-saint-christophe-des-bardes.com/

**Screenshots**

![Capture d’écran 2024-08-26 154230](https://github.com/user-attachments/assets/b53d9fb2-1dc5-4f27-b41d-b63db293e598)

 - Joomla Version: 5.1.13
 - PHP Version 8.2
 - Astroid Version Framework Version: 3.1.2
 -Template Version: 3.1.0
 - Device: laptop
 - OS:Win 11
 - Browser  Firefox, Chrome

**Comment by Chacapamac** [comment]
Maybe, its a design change choice?

**Comment by patrickus33** [comment]
No, absolutely not.
The change in the “blog” views occurred immediately after the update without any change on my part, either in the template or in the menu links.
I'm the only one with access to the site backend.

**Comment by sonvnn** [comment]
I am sorry about this inconvenience. I have update style of the One a bit. You can use custom css to override it. Please help me do this.

Thanks & Best regards,
Sonny

**Comment by patrickus33** [comment]
@sonvnn 
OK ! 
No problem, I've overrided the style with CSS.
Thank you for your explanation.

**Comment by patrickus33** [comment]
@sonvnn 
Rather than imposing a presentation, wouldn't it be better to propose it as an option in the “blog view configuration” section?
If you like, you could propose several.
That way, everyone would have a choice.

**Comment by sonvnn** [comment]
@patrickus33 As requested by some Astroid users, I tried to upgrade One to be compatible and use the new features of Astroid 3. As you know, One and Zero were released from Astroid 2 by another template provider, Joomdev. They are over 5 years old, and updating a template like that inevitably involves some changes in the HTML&CSS structure. 

I intended to forget about them and just focus on Two, but when there were many requests for using the new features of Astroid 3 on One and Zero. I had to upgrade it. I hope you sympathize and try to make changes to help me continue to support One and Zero. You know, it is really not easy to read another developer's code.

**Comment by patrickus33** [comment]
@sonvnn  I understand the difficulty of all this, but I hadn't considered this aspect of the problem.
In any case, thank you for this extraordinary work!
Taking over from Joomdev has delighted many Joomla! fans.
